src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -130,6 +140,103 @@ private bool TryAuthenticateFromCache(
 		return true;
 	}
 
+	private async Task<IOktaAuthenticatedClient?> TryAuthenticateWithDSSOAsync(
```

Comment:
This doesn't look idiomatic to me.
I'd expect a `Try...` method to return a Boolean or something like a "result" object. Using "nullness" to indicate failure isn't the best API design.

Reply:
nit: we don't have an agreed upon convention, but personally I'm of the opinion that acronyms & initialisms should be formatted like regular words in PascalCase & camelCase, i.e. `TryAuthenticateWithDssoAsync`.

See also names in .NET standard libraries:
[`HttpClient`](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=net-8.0), [`TcpClient`](https://learn.microsoft.com/en-us/dotnet/api/system.net.sockets.tcpclient?view=net-8.0), [`Dns`](https://learn.microsoft.com/en-us/dotnet/api/system.net.dns?view=net-8.0) [`AesGcm`](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.aesgcm?view=net-8.0), [`Utf8`](https://learn.microsoft.com/en-us/dotnet/api/system.text.unicode.utf8?view=net-8.0)